app/app.go

```diff
@@ -502,6 +502,9 @@ func (app *HeimdallApp) Name() string { return app.BaseApp.Name() }
 // InitChainer initializes chain
 func (app *HeimdallApp) InitChainer(ctx sdk.Context, req abci.RequestInitChain) abci.ResponseInitChain {
 	var genesisState GenesisState
+
+	json := jsoniter.ConfigCompatibleWithStandardLibrary
```

Comment:
Do we need a standard compatibility? Is it for all APIs or just bor<->heimdall? If the second option I'd rather use the fastest config.

Reply:
I left the most compatible choice overall.
Here the configs for fastest:
```
// ConfigFastest marshals float with only 6 digits precision
var ConfigFastest = Config{
	EscapeHTML:                    false,
	MarshalFloatWith6Digits:       true, // will lose precession
	ObjectFieldMustBeSimpleString: true, // do not unescape object field
}.Froze()
```

Wondering if this could break something in our APIs given the precision loss

Reply:
We just need tests from Bor side and ready to go. Also we can use the same package from Bor which will give us 100% compatibility.